config/development.json

```diff
@@ -39,8 +39,6 @@
 		"comments/filters-in-posts": true,
 		"comments/moderation-tools-in-posts": true,
 		"comments/management/threaded-view": false,
-		"create/focused-launch-flow": false,
```

Comment:
This flag shouldn't be deleted, as it is about enabling the Focused Launch modal in the iframed block editor — not the scope of this PR